### DISCUSSION TITLE
Where to download TTS models for Mandrain?

### DISCUSSION THREAD

**Discussion by lucasjinreal** [coqui-ai/TTS#1095] [General]
Where to download TTS models for Mandrain?

**Comment by erogol** [comment]
There is only one released model that was trained by a contributor. You can see it if you run

```tts --list_models ```

**Reply by lucasjinreal** [reply]
thanks, I found it `tts_models/zh-CN/baker/tacotron2-DDC-GST`
may I ask a very stupid question? If I have a tts model like this only in Chinese, can I using it combined with a vocoder trained on English to synasis voice?

**Reply by erogol** [reply]
you can try :)

**Reply by lucasjinreal** [reply]
I think it's not work
